app/assets/stylesheets/admin/views/_topical-events-index.scss

```diff
@@ -0,0 +1,20 @@
+.app-view-topical-events-index__table .govuk-table__row .govuk-table__header:nth-child(3) {
+  width: 21%;
+}
+
+.app-view-topical-events-index__table .govuk-table__row .govuk-table__header:nth-child(4) {
+  width: 15%;
+  text-align: center;
```

Comment:
I don't think the header should be aligned centre, just the column as you've done below.